twisher/src/main/java/posts/Post.java

```diff
@@ -0,0 +1,98 @@
+package posts;
+
+import java.util.Date;
+import java.util.List;
+
+public class Post {
+    public String id;
+    public String description;
+    public Date createdAt;
+    public String author;
+    public String photoLink;
+    private List<String> likes;
+
+    Post() {
+    }
+
+    public Post(String id, String description, Date createdAt, String author, String photoLink, List<String>likes) {
+        this.id = id;
+        this.description = description;
+        this.createdAt = createdAt;
+        this.author = author;
+        this.photoLink = photoLink;
+        this.likes = likes;
+    }
+
+    public Post(Post post) {
```

Comment:
When you support more than one constructor, it's better to arrange your code in such a way that one constructor calls another. Here it could be like `this(post.id, post.description, post.createdAt...)` etc.

But please remember that having more than 3 arguments in a method is a bad practice. No exclusion for constructor.
Use "builder" pattern instead, or pure setters